packages/test/test-end-to-end-tests/src/testSnapshotCache.ts

```diff
@@ -16,17 +16,20 @@ export interface ValueWithSnapshot {
 }
 
 export class TestSnapshotCache implements IPersistedCache {
-	private readonly cache = new Map<string, ValueWithSnapshot>();
+	private readonly cache = new Map<string, unknown>();
 	private readonly versionCache = new Map<string, ValueWithSnapshot>();
 	private readonly versionToCacheKey = new Map<string, string>();
-	public async get(entry: ICacheEntry): Promise<ValueWithSnapshot | undefined> {
+	public async get(entry: ICacheEntry): Promise<any | undefined> {
 		const key = getKeyForCacheEntry(entry);
 		return this.cache.get(key);
 	}
-	public async put(entry: ICacheEntry, value: ValueWithSnapshot): Promise<void> {
+	public async put(entry: ICacheEntry, value: any): Promise<void> {
 		const key = getKeyForCacheEntry(entry);
 		this.cache.set(key, value);
-		const versionKey = `${value.value.snapshotTree.id}`;
+		const versionKey = `${value.value?.snapshotTree?.id}`;
+		if (versionKey === undefined) {
```

Comment:
This condition will never be true because of the backtick template literal.

Reply:
Thanks for the catch!